### DISCUSSION TITLE
Debug strange crashes

### DISCUSSION THREAD

**Discussion by DearVa** [DearVa/ExplorerEx#28] [Announcements]
如果ExplorerEx在运行时出现闪退或者看不到界面程序就退出，且没有任何错误提示或者Log输出，可以下载下面的两个文件，将其放在软件的目录下，之后执行`RunWithDumpWatcher.bat`，如果程序出现问题就会生成类似`ExplorerEx.exe_220420_005130.dmp`的转储文件。请将该文件发送给我，这将协助我解决问题。非常感谢。

If ExplorerEx crashes or exits without seeing any interface, and there is no error prompt or log output, you can download the following two files, put them in the same directory of ExplorerEx, and then execute `RunWithDumpWatcher.bat`, if there is a problem with the program, it will generate a dump file like `ExplorerEx.exe_220420_005130.dmp`. Please send it to me, which will help me solve the problem. Thanks a lot!

<hr /><em>This discussion was created from the release <a href='https://github.com/DearVa/ExplorerEx/releases/tag/'>Debug strange crashes</a>.</em>

**Comment by TotoWang-hhh** [comment]
幸苦了作者，你已经在这个细节里暴露你1点钟还在写代码了……
![image](https://user-images.githubusercontent.com/67090055/164183960-89fc0b67-bd8d-431e-9113-43093ac0a94f.png)

**Reply by DearVa** [reply]
数媒大学生睡什么觉啊（恼）（bushi）
